nemo_curator/classifiers/fineweb_edu.py

```diff
@@ -43,7 +45,23 @@ def __init__(
         super().__init__(path_or_name=path_or_name, max_mem_gb=max_mem_gb)
 
     def load_model(self, device: str = "cuda"):
-        model = AutoModelForSequenceClassification.from_pretrained(self.path_or_name)
+        if self.path_or_name in [
+            FINEWEB_MIXTRAL_IDENTIFIER,
+            FINEWEB_NEMOTRON_IDENTIFIER,
+        ]:
+            model = AutoModelForSequenceClassification.from_pretrained(
+                self.path_or_name, torch_dtype=torch.bfloat16
+            )
```

Comment:
Interesting that we read `FINEWEB_MIXTRAL_IDENTIFIER `, `FINEWEB_NEMOTRON_IDENTIFIER ` in bfloat16 now. 

Do you know why we cant/dont do it for EDU classifier ? 

Can you add a comment stating the reason for this

Reply:
Not sure, this was how it was in the script that the Nemotron-CC developers used.

Reply:
Can we do a quick benchmark with autocast. I think this should happen automatically when we use `torch.autocast` , which we use/or should use. 

If the results on a dataset line up (for both accuracy and throughput) we can probably skip this fork.

Reply:
Just ran, the results look a bit different without the `bfloat16` but still pretty similar to the ones from before. I have removed `torch_dtype=torch.bfloat16` for now.